### DISCUSSION TITLE
Discovered way to access incognito mode when it's disabled on Chromebooks

### DISCUSSION THREAD

**Discussion by op-max** [3kh0/ext-remover#1141] [Exploits JOIN TITANIUM NETWORK TO SHARE PLEASE]
Although I never planned to share this, I wanted to because I don't fully understand how I can repeat it and to replicate the process easier. I have my wifi set to "connect automatically". Occasionally, I don't understand why, it remembers to access it and joins it, but asks me to sign in. Pressing the sign in button sends me to a corrupted Google API page where it asks to go back. Pressing "go back" and right clicking the window to "move to a new window" loads it in an incognito tab, or pressing different links will send you to an incognito tab with stack overflow detailing a discussion. I would like to understand how this works and how to replicate it.

Thanks, op-max

**Comment by op-max** [comment]
Also, any extensions are broken in incognito mode.

**Comment by S-PScripts** [comment]
images?

**Reply by op-max** [reply]
This way only works if your Chromebook's memory is to a point of literally corrupting other parts of ChromeOS. I got a new one, so I cannot reproduce this task.

I created another way that mirrored it. Unfortunately as of v127, it removed the option of "show as tab"; thus, the exploit cannot be done. 
**And as of v129, the incognito tab does not pull up.**
One note above. I do not think the version is the cause for this. It varies among other chromebooks. I have tested. If you do manage to get an incognito tab, it would either be in a form of a page similar to about:blank or a form that leads you to agree with the Public Wifi rules. 
Your best bet to convert the tab into a window is via the form.
I cannot say much on this topic. But this is all I understand of its properties.

**Reply by S-PScripts** [reply]
> This way only works if your Chromebook's memory is to a point of literally corrupting other parts of ChromeOS. I got a new one, so I cannot reproduce this task.
> 
> I created another way that mirrored it. Unfortunately as of v127, it removed the option of "show as tab"; thus, the exploit cannot be done. **And as of v129, the incognito tab does not pull up.** One note above. I do not think the version is the cause for this. It varies among other chromebooks. I have tested. If you do manage to get an incognito tab, it would either be in a form of a page similar to about:blank or a form that leads you to agree with the Public Wifi rules. Your best bet to convert the tab into a window is via the form. I cannot say much on this topic. But this is all I understand of its properties.

i created my own incognitos but they didn't work the same as yours https://s-pscripts.github.io/incognito-v123/

**Comment by SlitheringLizard1** [comment]
that is weird, usually Google API pages never redirect back to incognito. maybe the corruption is that instead of it being a normal tab it redirects to, it is incognito, and that is why the page is corrupted? idk I'm just spitballing
